cesium/internal/unary/db.go

```diff
@@ -87,9 +90,19 @@ func (db *DB) OpenIterator(cfg IteratorConfig) *Iterator {
 		Channel:        db.Channel,
 		internal:       iter,
 		IteratorConfig: cfg,
+		db:             db,
 	}
 	i.SetBounds(cfg.Bounds)
+	db.openIteratorWriters.Add(1)
 	return i
 }
 
+func (db *DB) TryClose() error {
+	if db.openIteratorWriters.Value() > 0 {
+		return errors.New("[unary] channel being written to")
```

Comment:
This error might confuse people because there may be open iterators and no open writers. Should probably say `channel has N iterators/writers accessing it.`